src/main/kotlin/org/jitsi/nlj/rtcp/RembHandler.kt

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright @ 2018 - present 8x8, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.jitsi.nlj.rtcp
+
+import org.jitsi.nlj.rtp.bandwidthestimation.BandwidthEstimator
+import org.jitsi.nlj.util.bps
+import org.jitsi.nlj.util.createChildLogger
+import org.jitsi.rtp.rtcp.RtcpPacket
+import org.jitsi.rtp.rtcp.rtcpfb.payload_specific_fb.RtcpFbRembPacket
+import org.jitsi.utils.logging2.Logger
+import java.util.concurrent.CopyOnWriteArrayList
+
+class RembHandler(parentLogger: Logger) : RtcpListener {
+    private val logger = parentLogger.createChildLogger(RembHandler::class)
+
+    private val bweUpdateListeners: MutableList<BandwidthEstimator.Listener> =
+        CopyOnWriteArrayList()
+
+    override fun rtcpPacketReceived(packet: RtcpPacket?, receivedTime: Long) {
+        if (packet is RtcpFbRembPacket) {
+            logger.debug { "Received REMB packet" }
+            onRembPacket(packet)
+        }
+    }
+
+    fun addListener(bweUpdateListener: BandwidthEstimator.Listener) {
+        bweUpdateListeners.add(bweUpdateListener)
+    }
+
+    private fun onRembPacket(rembPacket: RtcpFbRembPacket) {
+        logger.debug { "Updating bandwidth to ${rembPacket.bitrate.bps}" }
```

Comment:
Is there still a difference between `debug` and `cdebug`?

Reply:
No, in fact all the `c` prefix methods can go away now and we can just use this plain debug everywhere.  I'll add a card for that.

Reply:
Blah, actually (after reading my comment) they are _slightly_ different.  The `c` methods may be more efficient due to inlining, but honestly I'm not sure it's worth it to have the 2 different methods.